### PR TITLE
Jakarta EE  Web Profile 11

### PR DESCRIPTION
**Pull request by ivargrimstad**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] **N/A** Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] **N/A** Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

File: webprofile/11/_index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date: 2023-07-17
+summary: "Release of the Jakarta EE 11 Web Profile"
+---
+The Jakarta EE Web Profile defines a profile of the Jakarta EE Platform specifically targeted at web applications.
+
+### New features, enhancements or additions
+<!-- List here -->
+* Support for Java Records 
+* Support for Virtual Threads
+
```
Comment (edburns) [inline webprofile/11/_index.md:12]: Add 

```
* Jakarta Data 1.0
```

per https://dev.azure.com/jakarta-ee-azdo/jakarta-ee-azdo/_workitems/edit/171

**Review by edburns — CHANGES_REQUESTED** [review]
Add Jakarta Data 1.0 per https://dev.azure.com/jakarta-ee-azdo/jakarta-ee-azdo/_workitems/edit/171

**Comment by edburns** [comment]
See AB#171.

File: webprofile/11/_index.md (hunk 1 above)
Comment (Emily-Jiang) [inline webprofile/11/_index.md:12]:
```suggestion
* Jakarta Data 1.0
```

Comment (Emily-Jiang) [reply webprofile/11/_index.md]: @edburns done the change

**Review by edburns — APPROVED** [review]

**Pull request merged**